approval_polls/templates/approval_polls/poll_update_form.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class='row-fluid top-buffer'>
+        <div class='col-md-12'>
+            <h3> Edit Poll </h3>
+        </div>
+    </div>
+    <div class='row-fluid top-buffer'>
+        <div class='col-md-12'>
+            <form action="" method="post"> {% csrf_token %}
+                {{ form.non_field_errors }}
+                <div class="fieldWrapper">
+                    <label>Poll Active/Inactive:</label>
+                    {{ form.suspended }}
+                    {% if form.suspended.errors %}
```

Comment:
Can you follow the styles in the [create.html](https://github.com/electology/approval_frame/blob/master/approval_polls/templates/approval_polls/create.html#L17-L25) template for error reporting?